api/src/main/java/eu/europeana/fulltext/api/web/FTRetrievalController.java

```diff
@@ -94,6 +96,14 @@ private ResponseEntity<String> getAnnoPageInfo(String datasetId, String localId,
             return cached;
         }
         HttpHeaders headers = CacheUtils.generateHeaders(request, eTag, CacheUtils.zonedDateTimeToString(modified));
+        // by default content-type should be application/ld+json unless accept header is passed as application/json
+        boolean isJson = false;
+        if (request.getHeader(ACCEPT) != null && request.getHeader(ACCEPT).equals(MEDIA_TYPE_JSON)) {
+            isJson = true;
+        }
+        // add content-type. As no version is present in request hence will default to '2'
+        AcceptUtils.addContentTypeToResponseHeader(headers, null, isJson);
```

Comment:
Not entirely sure, but I think the manifest endpoint returns data in version 3, so we would need to sent always 3.

Reply:
done